catalog/core/catalog-core-migratable/src/test/java/org/codice/ddf/catalog/migratable/impl/MigrationFileWriterTest.java

```diff
@@ -210,6 +180,17 @@ public void testWriteMetacardsWriteObjectFails() throws Exception {
         List<Result> results = new ArrayList<>();
         results.add(new ResultImpl(new MetacardImpl()));
 
+        mockFile = new File("");
```

Comment:
Could throw an IOException in your mock object instead of making this a new File